docs/api/v3.rst

```diff
@@ -936,11 +1231,22 @@ Environment Variable delete
 
     **Example request**:
 
-    .. sourcecode:: bash
+   .. tabs::
```

Comment:
```suggestion
    .. tabs::
```

Reply:
sorry about that, must have missed the last one while checking. I fixed it.